crmsh/ui_context.py

```diff
@@ -248,8 +251,6 @@ def enter_level(self, level):
         self._in_transit = True
 
         entry = level()
-        if 'requires' in dir(entry) and not entry.requires():
-            self.fatal_error("Missing requirements")
```

Comment:
Currently, all sublevels will run this check.

And both `maintenance` and `configure` sublevel will call `cib_factory.initialize`, which will call cibadmin to query the CIB